doc/tasks/clover_coverage.md

```diff
@@ -0,0 +1,24 @@
+# Clover Coverage
+
+The Phpunit task will run your unit tests.
+It lives under the `clover_coverage` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        clover_coverage:
+            config_file: /tmp/clover.xml
+            level: 85
```

Comment:
This sample configuration always shows the default config in the other task files. Can you change the values?